client/main.lua

```diff
@@ -473,26 +452,23 @@ local function createVehicleZone(shopName, coords)
         coords = coords.xyz,
         size = shop.zone.size,
         rotation = coords.w,
-        debug = shop.zone.debug,
-        onEnter = enteringVehicleSellZone,
-        inside = insideVehicleSellZone,
+        debug = config.debugPoly,
+        onEnter = function()
+            local job = config.shops[insideShop].job
+            if not QBX.PlayerData or not QBX.PlayerData.job or (QBX.PlayerData.job.name ~= job and job ~= 'none') then return end
```

Comment:
```suggestion
            if QBX.PlayerData.job.name ~= job then return end
```
I think we can/should assume that QBX.PlayerData exists. I also believe it's part of the spec that job is required, not optional.